### PR TITLE
Redesign Forgot Password Page

### PR DESCRIPTION
**Pull request by loic-sharma**
Updates the following pages:

* https://localhost/account/ForgotPassword
* https://localhost/account/PasswordSent

Related to #4017

### CODE REVIEW

**Comment by loic-sharma** [comment]
Ready for review @microsoftsam :)

File: src/NuGetGallery/Views/Users/ForgotPassword.cshtml [hunk 1]
```diff
@@ -1,28 +1,54 @@
...
+    {
+            <span class="help-block">@error.ErrorMessage</span>
+    }
+}
+}
 
-        @Html.AntiForgeryToken()
-        @Html.ValidationSummary(true)
+<div class="container main-container page-forgot-password">
+    <div class="row">
```
Comment (microsoftsam) [inline src/NuGetGallery/Views/Users/ForgotPassword.cshtml]: No need for a row here. You'd lose the padding around the container this way, and unless you're using more than one column (or col offsets), there's no need for a row. 

To help explain what I'm talking about. Here's what it looks like right now on a small screen: 
screen shot 2017-06-01 at 2 51 01 pm
(The row in this case offsets the padding added by container, and the text hits the edges)

Without a row: 
screen shot 2017-06-01 at 2 50 54 pm
(Things are centered, and text has a little padding off the sides).

File: src/NuGetGallery/Views/Users/ForgotPassword.cshtml [hunk 2]
```diff
@@ -1,28 +1,54 @@
...
+        <h1>Forgot Password</h1>
+    </div>
 
-        @Html.EditorForModel()
+    <div class="row">
```
Comment (microsoftsam) [inline src/NuGetGallery/Views/Users/ForgotPassword.cshtml]: Same here, remove the row div.

File: src/NuGetGallery/Views/Users/ForgotPassword.cshtml [hunk 3]
```diff
@@ -1,28 +1,54 @@
...
+        <p>We are sorry to hear you forgot your NuGet.org account password. Enter your email below and we will send instructions to reset your password.</p>
+    </div>
 
-        <img src="@Url.Content("~/Content/images/required.png")" alt="Blue border on left means required." />
+    <div class="row forgot-form">
+        <div class="col-md-offset-3 col-md-6">
+            @using (Html.BeginForm())
```
Comment (microsoftsam) [inline src/NuGetGallery/Views/Users/ForgotPassword.cshtml]: You can probably afford to have the offset apply for the sm view too here. Use col-sm-offset-3 and col-sm-6 instead

File: src/NuGetGallery/Views/Users/PasswordSent.cshtml [hunk 4]
```diff
@@ -1,16 +1,28 @@
...
-<p>
-    We've sent you an email containing a temporary url that will allow you to reset your NuGet.org
-    account password for the next @ViewBag.Expiration hours.
-</p>
-<p>
-    Please check your spam folder if you don't receive the email within a
-    few minutes.
-</p>
\ No newline at end of file
+    <div class="row">
```
Comment (microsoftsam) [inline src/NuGetGallery/Views/Users/PasswordSent.cshtml]: Same here, no row.

**Comment by loic-sharma** [comment]
Updated with fixes for the issues found by Sam.

File: src/NuGetGallery/Views/Users/ForgotPassword.cshtml [hunk 5]
```diff
@@ -1,28 +1,54 @@
...
-        <input type="submit" value="Send" title="Send Instructions" />
-        <a class="cancel" href="@Url.LogOn()" title="Cancel Changes and go back.">Cancel</a>
-    </fieldset>
-}
\ No newline at end of file
+                <div class="form-group @HasError("Email")">
+                    @Html.LabelFor(m => m.Email)
+                    @Html.TextBoxFor(m => m.Email, new { @class = "form-control" })
+                    @ShowErrors("Email")
+                </div>
```
Comment (microsoftsam) [inline src/NuGetGallery/Views/Users/ForgotPassword.cshtml]: Need to add an aria-describedby="helptextID" for the help text that shows up in the event of an error.

Comment (loic-sharma) [inline src/NuGetGallery/Views/Users/ForgotPassword.cshtml]: Good catch. This was fixed in the latest commit.

File: src/NuGetGallery/Views/Users/ForgotPassword.cshtml [hunk 6]
```diff
@@ -1,28 +1,50 @@
...
-    </fieldset>
-}
\ No newline at end of file
+                <div class="form-group @HasError("Email")">
+                    @Html.LabelFor(m => m.Email, new { id = "email-label" })
+                    @Html.TextBoxFor(m => m.Email, new { @class = "form-control", aria_labelledby = "email-label email-error-description" })
+                    @ShowErrors("Email")
+                </div>
+                <div class="form-group">
+                    <input type="submit" class="btn btn-primary form-control" value="Send" />
```
Comment (microsoftsam) [inline src/NuGetGallery/Views/Users/ForgotPassword.cshtml]: Add aria-required="true" to indicate that the field is required.

**Pull request merged**